tests/fast_tests/test_rewards.py

```diff
@@ -29,12 +32,27 @@ def test_desired_velocity(self):
         # check the average speed upon reset
         self.assertEqual(desired_velocity(env, fail=False), 0)
 
+        # check the average speed upon reset with a subset of edges
+        self.assertEqual(desired_velocity(env, edge_list=["bottom"],
+                                          fail=False), 0)
+
         # change the speed of one vehicle
         env.vehicles.test_set_speed("test_0", 10)
 
         # check the new average speed
         self.assertAlmostEqual(desired_velocity(env, fail=False), 0.05131670)
```

Comment:
Can you pick a number such that this is basically an integer? For example, put just one or two vehicles into the system. It just seems cleaner and then obvious when something is wrong.